### PR TITLE
Create New Monitor cleanup

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Create New Monitor cleanup to prepare for [metrics anomaly release](https://github.com/SumoLogic/sumologic-documentation/pull/4577)

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-453

### CODE REVIEW

File: docs/alerts/monitors/create-monitor.md [hunk 1]
```diff
@@ -5,122 +5,146 @@ description: Learn how to create a Sumo Logic monitor.
...
-1. In the **Trigger Type** section of the monitor editor, enable the checkbox that corresponds to the threshold value that you want to use (either "Critical", "Warning", or both).
-1. The threshold values will be the same as defined in the Metrics page for both Critical and Warning thresholds.
-1. All other parameters should be set to default, including the window (15 minutes) and the "at all times" box.
-1. Ensure that the Recover value is set to the default, which is the opposite of the Alert value. The Edit Recovery button should be off.
-1. Once all values have been set, click on **Save** to create the monitor.
+1. Open the Metrics Explorer page:
+   * [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). From Sumo Logic home, click **Metrics**.<br/><img src={useBaseUrl('img/monitors/home-metrics.png')} alt="Screenshot of the Sumo Logic home page with rectangle around the Metrics icon" width="200"/>
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:45]:
```suggestion
   * [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). From Sumo Logic home, click **Metrics**.<br/><img src={useBaseUrl('img/monitors/home-metrics.png')} alt="Screenshot of the Sumo Logic home page with rectangle around the Metrics icon" style={{border: '1px solid gray'}} width="200"/>
```

File: docs/alerts/monitors/create-monitor.md [hunk 2]
```diff
@@ -5,122 +5,146 @@ description: Learn how to create a Sumo Logic monitor.
...
+1. Click the three-dot kebab icon button at the end of the query field and select **Create a Monitor**.<br/> <img src={useBaseUrl('img/monitors/create-monitor.png')} alt="Screenshot of the Metrics Explorer in Sumo Logic, showing the dropdown menu accessed via the three vertical dots icon. The menu includes options for Basic Mode, Duplicate Query, Create a Monitor, and Create an SLO. The option 'Create a Monitor' is highlighted. Below the menu, the thresholds panel shows critical and warning thresholds set to 500000000 and 80, respectively, with the 'Fill remaining area as green' option toggled off." width="400"/>
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:49]:
```suggestion
1. Click the three-dot kebab icon button at the end of the query field and select **Create a Monitor**.<br/> <img src={useBaseUrl('img/monitors/create-monitor.png')} alt="Screenshot of the Metrics Explorer in Sumo Logic, showing the dropdown menu accessed via the three vertical dots icon. The menu includes options for Basic Mode, Duplicate Query, Create a Monitor, and Create an SLO. The option 'Create a Monitor' is highlighted. Below the menu, the thresholds panel shows critical and warning thresholds set to 500000000 and 80, respectively, with the 'Fill remaining area as green' option toggled off." style={{border: '1px solid gray'}} width="400"/>
```

File: docs/alerts/monitors/create-monitor.md [hunk 3]
```diff
@@ -5,122 +5,146 @@ description: Learn how to create a Sumo Logic monitor.
...
-1. The threshold values will be the same as defined in the Metrics page for both Critical and Warning thresholds.
-1. All other parameters should be set to default, including the window (15 minutes) and the "at all times" box.
-1. Ensure that the Recover value is set to the default, which is the opposite of the Alert value. The Edit Recovery button should be off.
-1. Once all values have been set, click on **Save** to create the monitor.
+1. Open the Metrics Explorer page:
+   * [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). From Sumo Logic home, click **Metrics**.<br/><img src={useBaseUrl('img/monitors/home-metrics.png')} alt="Screenshot of the Sumo Logic home page with rectangle around the Metrics icon" width="200"/>
+   * [**New UI**](/docs/get-started/sumo-logic-ui). Click the **Go To...** menu at the top of the screen and select **Metrics Search**.
+1. Enter a metrics query. For example:<br/><img src={useBaseUrl('img/monitors/metrics-explorer-view.png')} alt="Screenshot of the Metrics Explorer in Sumo Logic displaying a query" width="500"/>
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:47]:
```suggestion
1. Enter a metrics query. For example:<br/><img src={useBaseUrl('img/monitors/metrics-explorer-view.png')} alt="Screenshot of the Metrics Explorer in Sumo Logic displaying a query" style={{border: '1px solid gray'}} width="500"/>
```

File: docs/alerts/monitors/create-monitor.md [hunk 4]
```diff
@@ -5,122 +5,146 @@ description: Learn how to create a Sumo Logic monitor.
...
+1. In the **Thresholds** section, define the critical and warning thresholds for your metrics query.<br/> <img src={useBaseUrl('img/monitors/metrics-explorer-thresholds.png')} alt="Screenshot of the Metrics Explorer in Sumo Logic, displaying a line chart for node memory utilization over time. The chart shows the memory utilization metric from 17:42:12 to 17:57:12 on 21/02/2023. The right side of the screen includes a thresholds panel with critical and warning thresholds set to 500000000 and 80, respectively. The 'Fill remaining area as green' option is toggled off." width="600"/>
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:48]:
```suggestion
1. In the **Thresholds** section, define the critical and warning thresholds for your metrics query.<br/> <img src={useBaseUrl('img/monitors/metrics-explorer-thresholds.png')} alt="Screenshot of the Metrics Explorer in Sumo Logic, displaying a line chart for node memory utilization over time. The chart shows the memory utilization metric from 17:42:12 to 17:57:12 on 21/02/2023. The right side of the screen includes a thresholds panel with critical and warning thresholds set to 500000000 and 80, respectively. The 'Fill remaining area as green' option is toggled off." style={{border: '1px solid gray'}} width="600"/>
```

File: docs/alerts/monitors/create-monitor.md [hunk 5]
```diff
@@ -5,122 +5,146 @@ description: Learn how to create a Sumo Logic monitor.
...
+1. The **New Monitor** will open with prefilled data based on the threshold values you set in the previous steps.<br/> <img src={useBaseUrl('img/monitors/new-monitor-window.png')} alt="Screenshot of the 'New Monitor' setup page in Sumo Logic, specifically focusing on the Trigger Conditions section. The Monitor Type is set to Metrics and Detection Method to Static. The query is set for node memory utilization for a specific collector. The Alert Grouping options include one alert per monitor or one alert per time series. The Trigger Type section shows critical alerts set to trigger when the result is greater than or equal to 500000000 within 15 minutes. The recovery settings are enabled to recover automatically when the result is less than 500000000 within a 15-minute window. Historical Trend is displayed below, with a dashed red line indicating the threshold." width="600"/>
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:50]:
```suggestion
1. The **New Monitor** will open with prefilled data based on the threshold values you set in the previous steps.<br/> <img src={useBaseUrl('img/monitors/new-monitor-window.png')} alt="Screenshot of the 'New Monitor' setup page in Sumo Logic, specifically focusing on the Trigger Conditions section. The Monitor Type is set to Metrics and Detection Method to Static. The query is set for node memory utilization for a specific collector. The Alert Grouping options include one alert per monitor or one alert per time series. The Trigger Type section shows critical alerts set to trigger when the result is greater than or equal to 500000000 within 15 minutes. The recovery settings are enabled to recover automatically when the result is less than 500000000 within a 15-minute window. Historical Trend is displayed below, with a dashed red line indicating the threshold." style={{border: '1px solid gray'}} width="600"/>
```

File: docs/alerts/monitors/create-monitor.md [hunk 6]
```diff
@@ -5,122 +5,146 @@ description: Learn how to create a Sumo Logic monitor.
...
+
+In the next step, you'll need to provide a logs or metrics query.
+
+:::note logs and metrics monitors only
+No need to enter a query for **SLO** monitors.
+:::
+
+**Logs** monitors can have one query up to 15,000 characters long.
+
+**Metrics** monitors can have up to six queries. When providing multiple metrics queries, use the letter labels to reference a query row. The monitor will automatically detect the query that triggers your alert, and will mark that row with a notification bell icon. See [Joined metrics queries](/docs/metrics/metrics-queries/metrics-explorer/#join-metric-queries) for details.<br/> ![Screenshot of the 'New Monitor' setup page in Sumo Logic, showing the Trigger Conditions section. Metrics is selected as the Monitor Type and Static as the Detection Method. The query includes two metrics: CPU_Sys and CPU_User, with an alert condition combining both metrics (#B + #C). A bell icon is highlighted on the left side.](/img/monitors/metrics-monitor-query-row.png)
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:119]: Put the screenshot into the new image tagging.

File: docs/alerts/monitors/create-monitor.md [hunk 7]
```diff
@@ -5,122 +5,146 @@ description: Learn how to create a Sumo Logic monitor.
...
+* A numeric field returned from the search. You can pick any numeric field from your query, and alert on the value of that field. The field is `_count` in the above screenshot. To convert a string to a number use the [`num` operator](/docs/search/search-query-language/search-operators/num). For example, if you have a field named **duration**, you would use the `num` operator as follows to convert it to a number value.
+   ```sh
+   | num(duration)
+   ```
+
+#### Static detection method
+
+**Example: Log Trigger Type - Critical and Warning**  
 
 ![logs trigger type.png](/img/monitors/logs-trigger-type.png)
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:149]: Put the screenshot into the new image tagging.

File: docs/alerts/monitors/create-monitor.md [hunk 8]
```diff
@@ -130,17 +154,19 @@ You can set the trigger based on the following:
...
+For example, when the alert is set to `> 10` the recovery would be set to `<= 10` when inferred. Sumo Logic automatically resolves the incident when the resolution condition is satisfied.
+
+`Recover automatically when result is <threshold type> <threshold> for the selected time period`.
 
-##### Configurable Resolution Window
+#### Configurable Resolution Window
 
 When configuring monitor trigger conditions, you can set a resolution window to quickly resolve alerts when the underlying issues are fixed. This controls how long a monitor will wait prior to resolving the alert, when the underlying issues was corrected. For example, if your monitor is evaluating the last 60 minutes, you can specify a resolution window of 15 minutes. Once 15 minutes has elapsed with your monitor resolution window continuously satisfied, the alert will resolve. <br/>![config-resolution-window](/img/monitors/config-resolution-window-2.png)
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:171]: The sceenshot needs a border. Put the screenshot into the new image tagging.

File: docs/alerts/monitors/create-monitor.md [hunk 9]
```diff
@@ -130,17 +154,19 @@ You can set the trigger based on the following:
...
 | Threshold type | How you want the value compared. Select either **greater than**, **greater than or equal**, **less than or equal**, or **less than**. |
 | Threshold | The value against which the trigger will be evaluated. You can specify any valid numeric value up to **1,000**. |
-| Time range | The duration of time to evaluate. Select either **5 minutes**, **10 minutes**, **15 minutes**, **30 minutes**, **1 hour**, **3 hours**, **6 hours**, **12 hours**, or **24 hours**. |
+| Time range | The duration of time to evaluate. Select **5 minutes**, **10 minutes**, **15 minutes**, **30 minutes**, **1 hour**, **3 hours**, **6 hours**, **12 hours**, or **24 hours**. |
 
 **Recover**
 
 Recovery condition is set by default to the opposite of the alert condition. If you need to change these settings, first switch the **Edit Recovery Settings** toggle and then adjust values for the recovery settings accordingly.
 
 ![logs trigger recovery toggle.png](/img/monitors/edit-recovery-settings1.png)
```
Comment (jpipkin1) [inline docs/alerts/monitors/create-monitor.md:163]: The sceenshot needs a border. Put the screenshot into the new image tagging.

File: docs/get-started/ai-machine-learning.md [hunk 10]
```diff
@@ -63,7 +63,7 @@ LogCompare simplifies log analysis by enabling easy comparison of log data from
 
 #### Anomaly Detection
 
-[Anomaly Detection](/docs/alerts/monitors/create-monitor/#select-monitor-type-and-detection-method), powered by machine learning, efficiently flags suspicious activities by establishing baseline behavior and minimizing false positives. It also automatically fine-tunes anomaly detection with minimal user input, and you can associate it with a playbook to link anomaly responses with monitors, streamlining incident response.
+Anomaly Detection, powered by machine learning, efficiently flags suspicious activities by establishing baseline behavior and minimizing false positives. It also automatically fine-tunes anomaly detection with minimal user input, and you can associate it with a playbook to link anomaly responses with monitors, streamlining incident response. [Learn more](/docs/alerts/monitors/create-monitor/#step-1-set-trigger-conditions).
```
Comment (jpipkin1) [inline docs/get-started/ai-machine-learning.md:66]: If you're going to use "Learn more" here, I suggest you use the same link approach  in all the rest of the entries in this article.

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
Thanks for cleaning up these articles! I have just a few suggestions.

File: docs/get-started/ai-machine-learning.md (hunk 10 above)
Comment (kimsauce) [reply docs/get-started/ai-machine-learning.md:66]: thank you! Just reverted this.

**Pull request merged**